### PR TITLE
Added a build profile for use with kafka version 0.10.0.1

### PR DESCRIPTION
**Pull request by IvanAtanasov**
the default of 0.10.2.0 is not working with our 0.10.0.1 kafka cluster. It produces java.nio.channels.ClosedChannelException exceptions whenever messages are added to be consumed

If there is a better solution than this, I am happy to use it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -657,5 +657,24 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>kafka-0.10.0.1</id>
```
Comment (HenryCaiHaiying) [inline pom.xml]: This version is actually lower than kafka-0.10.dev where it was using kafka 0.10.2.0, is there a particular reason that you need to use 0.10.0.1?

Comment (IvanAtanasov) [reply pom.xml]: I can't remember the specific errors but when trying to consume from a 0.10.0.1 cluster it does not work.

**Comment by HenryCaiHaiying** [comment]
We probably prefer to keep less version profiles in the pom file, especially 0.10.0.1 is a deprecated version.

**Comment by artem-garmash** [comment]
We have to do the same to make secor work with 0.10.0.1 clusters. The problem was in ZookeeperConnector. It was failing to fetch offsets and consuming would never start. The error message was:
```
Error while fetching offsets from 10.0.1.2:9092. Possible cause: null
```
While in kafka logs, there was the error about invalid version:
```
kafka.network.InvalidRequestException: Error getting request for apiKey: 9 and apiVersion: 2
        at kafka.network.RequestChannel$Request.liftedTree2$1(RequestChannel.scala:95)
        at kafka.network.RequestChannel$Request.<init>(RequestChannel.scala:87)
        at kafka.network.Processor$$anonfun$processCompletedReceives$1.apply(SocketServer.scala:488)
        at kafka.network.Processor$$anonfun$processCompletedReceives$1.apply(SocketServer.scala:483)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
        at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
        at kafka.network.Processor.processCompletedReceives(SocketServer.scala:483)
        at kafka.network.Processor.run(SocketServer.scala:413)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: Invalid version for API key 9: 2
        at org.apache.kafka.common.protocol.ProtoUtils.schemaFor(ProtoUtils.java:31)
        at org.apache.kafka.common.protocol.ProtoUtils.requestSchema(ProtoUtils.java:44)
        at org.apache.kafka.common.protocol.ProtoUtils.parseRequest(ProtoUtils.java:60)
        at org.apache.kafka.common.requests.OffsetFetchRequest.parse(OffsetFetchRequest.java:117)
        at org.apache.kafka.common.requests.AbstractRequest.getRequest(AbstractRequest.java:52)
        at kafka.network.RequestChannel$Request.liftedTree2$1(RequestChannel.scala:92)
```
Better way to handle this would be to add kafka version property to the pom to allow easy version override when building secor.

**Comment by HenryCaiHaiying** [comment]
OK, I will incorporate this change.

**Pull request merged**